pkg/volume/vsphere_volume/vsphere_volume.go

```
@@ -119,11 +119,15 @@ func (plugin *vsphereVolumePlugin) newUnmounterInternal(volName string, podUID t
 }
 
 func (plugin *vsphereVolumePlugin) ConstructVolumeSpec(volumeName, mountPath string) (*volume.Spec, error) {
+	mounter := plugin.host.GetMounter()
+	pluginDir := plugin.host.GetPluginDir(plugin.GetPluginName())
+	volumePath, _ := mounter.GetDeviceNameFromMount(mountPath, pluginDir)
```

Why is the error swallowed here?